taskfiles/utils.yml

```diff
@@ -103,3 +103,35 @@ tasks:
         . "{{.OUTPUT_DIR}}/bin/activate"
         pip3 install --upgrade pip
         pip3 install --upgrade -r "{{.REQUIREMENTS_FILE}}"
+
+  # ===
+  # clang UTILS
+  # ===
+  clang-format:
```

Comment:
Let's add a docstring to both clang-format and clang-tidy like we have for the checksum utils.